sdk/appcomplianceautomation/Azure.ResourceManager.AppComplianceAutomation/src/Generated/AppComplianceReportSnapshotResource.cs

```diff
@@ -180,28 +180,28 @@ public virtual Response<SnapshotResource> Get(CancellationToken cancellationToke
         /// </item>
         /// <item>
         /// <term>Default Api Version</term>
-        /// <description>2022-11-16-preview</description>
+        /// <description>2024-06-27</description>
         /// </item>
         /// <item>
         /// <term>Resource</term>
-        /// <description><see cref="SnapshotResource"/></description>
+        /// <description><see cref="AppComplianceReportSnapshotResource"/></description>
         /// </item>
         /// </list>
         /// </summary>
         /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return after starting the operation. For more information on long-running operations, please see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md"> Azure.Core Long-Running Operation samples</see>. </param>
         /// <param name="content"> Parameters for the query operation. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="content"/> is null. </exception>
-        public virtual async Task<ArmOperation<DownloadResponse>> DownloadAsync(WaitUntil waitUntil, SnapshotDownloadContent content, CancellationToken cancellationToken = default)
+        public virtual async Task<ArmOperation<AppComplianceDownloadResult>> DownloadAsync(WaitUntil waitUntil, SnapshotDownloadContent content, CancellationToken cancellationToken = default)
```

Comment:
Parameter name `SnapshotDownloadContent` is quite confusing, better to rename to `SnapshotDownloadRequestContent`

Reply:
fixed